recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -1,19 +1,20 @@
 --- a/CMakeLists.txt
 +++ b/CMakeLists.txt
-@@ -1,14 +1,14 @@
+@@ -1,14 +1,15 @@
 -project(basisu)
 -
  cmake_minimum_required(VERSION 3.0)
 -option(BUILD_X64 "build 64-bit" TRUE)
 -option(STATIC "static linking" FALSE)
 -option(SSE "SSE 4.1 support" FALSE)
 +
-+project(libbasisu VERSION 1.15.0 LANGUAGES CXX)
++project(basisu VERSION 1.15.0 LANGUAGES CXX)
 +
 +option(SSE4 "SSE4 4.1 support" FALSE)
  option(ZSTD "ZSTD support for KTX2 transcoding/encoding" TRUE)
 +option(BASISU_NO_ITERATOR_DEBUG_LEVEL "Change Iterator debug level" FALSE)
```

Comment:
unused?

```suggestion
```

Reply:
That was a mistake, now it's correctly used I believe.